scripts/front_end/README.md

```diff
@@ -20,21 +20,38 @@ mvn package
 cd scripts/front_end/trias-oauth/oauth-server/src/main/resources/db/
 mysql -u trias -p trias
 source trias_server-init.sql
+cd scripts/front_end/trias-oauth/oauth-server/src/main/resources/db/2019-05-28/
```

Comment:
hard code time?

Reply:
@niming2100  Need to modify the name of the folder